api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiProperties.java

```diff
@@ -0,0 +1,58 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.APIException;
+import org.openmrs.module.registrationcore.RegistrationCoreConstants;
+import org.openmrs.module.registrationcore.api.ModuleProperties;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class MpiProperties extends ModuleProperties {
+
+    public String getServerUrl() {
+        String propertyName = RegistrationCoreConstants.GP_MPI_URL;
+        return getProperty(propertyName);
+    }
+
+    public Integer getGlobalIdentifierDomainId() {
+        String propertyName = RegistrationCoreConstants.GP_MPI_GLOBAL_IDENTIFIER_DOMAIN_ID;
+        return getIntegerProperty(propertyName);
+    }
+
+    public Integer getMpiPersonIdentifierTypeId() {
+        String propertyName = RegistrationCoreConstants.GP_MPI_PERSON_IDENTIFIER_ID;
+        return getIntegerProperty(propertyName);
+    }
+
+    public MpiCredentials getMpiCredentials() {
```

Comment:
Ok, so we can make combination of username and password not general but just for OpenEmpi for now. 
Am I right?